saleor/search/backends/dashboard.py

```diff
@@ -0,0 +1,135 @@
+from . import elasticsearch2
```

Comment:
@patrys Dashboard custom backend is based on elasticsearch2 backend, it should be compatibile with elasticsearch5 as well. DB backend is not supported for dashboard search.